crates/sui-protocol-config/src/lib.rs

```diff
@@ -2179,8 +2179,8 @@ impl ProtocolConfig {
 
                     // Also bumps framework snapshot to fix binop issue.
 
-                    // enable bridge in devnet
-                    if chain != Chain::Mainnet && chain != Chain::Testnet {
+                    // enable bridge in devnet and testnet
+                    if chain != Chain::Mainnet {
```

Comment:
@mystenmark  we can't do this in a cherry pick since v45 is already on testnet. This needs to be in v46, right?

Reply:
Correct. You'll need to enable it in v46 for testnet.